roles/basic_setup/tasks/main.yml

```diff
@@ -24,3 +24,12 @@
       group: root
       mode: "0755"
     tags: umask
+    
+- name: Change file permissions of KDE document templates
+  file:
+    path: /usr/share/templates
+    recurse: yes
+    owner: root
+    group: root
+    mode: '0664'
```

Comment:
Warum mode: 0664, das bedeutet ja rw für user: root und rw für group root (in der ja keiner drinn ist) und dann r für other. Im Moment ist es für die Dateien: 0644 also für den Normalen User auch nur lesbar. Macht keinen unterscheid, oder?

Wenn du das laufen lässt, wird vermutlich auch das Directory so gesetzt und dann kann man das ja nicht mehr mit cd rein und den Inhalt sehen, das ist ja vermutlich nicht, was du willst.